pkg/monitoring/metrics/common/client/latency_adapter.go

```diff
@@ -31,5 +32,29 @@ type latencyAdapter struct {
 }
 
 func (l *latencyAdapter) Observe(_ context.Context, verb string, u url.URL, latency time.Duration) {
-	l.m.WithLabelValues(verb, u.String()).Observe(latency.Seconds())
+	l.m.WithLabelValues(determineVerb(u, verb), u.String()).Observe(latency.Seconds())
```

Comment:
can we use the `parseURLResourceOperation` instead of re-writing a similar function here?

```
    _, verb := parseURLResourceOperation(u)
    l.m.WithLabelValues(verb, u.String()).Observe(latency.Seconds())
```

IIUC the arg of parseURLResourceOperation will have to be changed but might be better to consolidate on the logic

Reply:
Updated, PTAL @alaypatel07